app/bibleview-js/src/components/TextEditor.vue

```diff
@@ -51,9 +52,18 @@ export default {
   emits: ['save', "close"],
   setup(props, {emit}) {
     const android = inject("android");
+    const features = inject('customFeatures')
     const editorElement = ref(null);
     const editor = ref(null);
     const inputText = ref(null);
+    var languages = null
```

Comment:
```suggestion
    const languages = null
```
(let or const as of es6 javascript, not old-style vars)